<ui version="4.0" >
 <class>ParagraphUi</class>
 <widget class="QWidget" name="ParagraphUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>176</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>5</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>222</width>
    <height>16</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QCheckBox" name="indentCB" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>&amp;Indent Paragraph</string>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="0" column="1" >
      <widget class="QGroupBox" name="aligmentGB" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Alignment</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="4" column="0" >
         <widget class="QRadioButton" name="alignRightRB" >
          <property name="text" >
           <string>&amp;Right</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QRadioButton" name="alignCenterRB" >
          <property name="text" >
           <string>&amp;Center</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QRadioButton" name="alignLeftRB" >
          <property name="text" >
           <string>&amp;Left</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QRadioButton" name="alignJustRB" >
          <property name="text" >
           <string>&amp;Justified</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QRadioButton" name="alignDefaultRB" >
          <property name="font" >
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="toolTip" >
           <string>Use the default alignment for this paragraph, whatever it is.</string>
          </property>
          <property name="text" >
           <string>Paragraph's &amp;Default</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QGroupBox" name="labelwidthGB" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Label Width</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="TextLabel2" >
          <property name="toolTip" >
           <string>This text defines the width of the paragraph label</string>
          </property>
          <property name="text" >
           <string>&amp;Longest label</string>
          </property>
          <property name="buddy" >
           <cstring>labelWidth</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="labelWidth" >
          <property name="toolTip" >
           <string>This text defines the width of the paragraph label</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>Line &amp;spacing</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QComboBox" name="linespacing" >
          <item>
           <property name="text" >
            <string>Default</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Single</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Double</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Custom</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="linespacingValue" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item rowspan="2" row="1" column="2" >
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QCheckBox" name="synchronizedViewCB" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>0</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>I&amp;mmediate Apply</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="applyPB" >
         <property name="text" >
          <string>&amp;Apply</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="restorePB" >
         <property name="text" >
          <string>&amp;Restore</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>linespacing</tabstop>
  <tabstop>linespacingValue</tabstop>
  <tabstop>indentCB</tabstop>
  <tabstop>labelWidth</tabstop>
  <tabstop>restorePB</tabstop>
  <tabstop>applyPB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_helpers.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
